Source/AwsCommonRuntimeKit/http/HTTPMonitoringOptions.swift

```diff
@@ -10,8 +10,10 @@ public struct HTTPMonitoringOptions: CStruct {
     /// Must be at least two
     public var allowableThroughputFailureInterval: UInt32
 
-    init(minThroughputBytesPerSecond: UInt64 = 0,
-         allowableThroughputFailureInterval: UInt32 = 2) {
+    public init(
+        minThroughputBytesPerSecond: UInt64 = 1,
+        allowableThroughputFailureInterval: UInt32 = 2
```

Comment:
Is 2 too low? What's the default in Java SDK?

Reply:
No default, I have removed it here as well.